rgw/v2/utils/utils.py

```diff
@@ -263,3 +263,12 @@ def get_hostname_ip():
 def cmp(val1, val2):
     return (val1 > val2) - (val1 < val2)
 
+def get_ceph_version():
+    """
+    get the current ceph version
+    """
+    log.info('get ceph version')
+    ceph_version= exec_shell_cmd('sudo ceph version')
+    version_info = ceph_version.split()[4]
+    return (version_info)
```

Comment:
you are returning a tuple here which has a single element in it. 
this would be more useful `version_id, version_name = version_info[2], version_info[4]`
and do `return version_id, version_name`

Reply:
Hi Rakesh,
Incorporated the changes. 
Please find the logs at http://magna002.ceph.redhat.com/ceph-qe-logs/madhavi/PR_logs/PR_75/verify_num_shards.with_review.console.log